### PR TITLE
Use JavascriptCatalog view if Django version is >= 2.0

### PR DESCRIPTION
**Pull request by aminland**
Deprecated since version 1.10:
javascript_catalog() is deprecated in favor of JavaScriptCatalog and will be removed in Django 2.0.

### CODE REVIEW

File: jet/urls.py [hunk 1]
```diff
@@ -24,13 +24,23 @@
         model_lookup_view,
         name='model_lookup'
     ),
-    url(
+]
+
+if django.VERSION[0] >= 2:
+    from django.views.i18n import JavascriptCatalog
```
Comment (soloidx) [inline jet/urls.py]: I have a typo error with the django 2.0 version I think it should be "JavaScriptCatalog"

**Comment by aminland** [comment]
@soloidx true that. fixed it (i think)

**Comment by SalahAdDin** [comment]
@aminland Can you make this PR in `jet_community

**Comment by aminland** [comment]
closing in favor of #265

**Pull request closed without merging**